core/sql/regress/hive/EXPECTED015

```diff
@@ -1400,10 +1397,6 @@ Task:  POPULATE INDEX  Status: Ended      ET: 00:00:12.244
 
 --- SQL operation complete.
 >>
->>cqd comp_bool_226 'on' ;
```

Comment:
Does this mean that we are not testing with TRAF_ALLOW_RISKY_INDEX_MAINTENANCE 'ON'